### PR TITLE
Do not restrict IDs to numbers.

### PR DESCRIPTION
**Pull request by d-ashfield**
#PATCH#

# CHANGELOG

Remove restriction on IDs to be numbers when using queryForCurrentValue.

### CODE REVIEW

**Comment by sandersky** [comment]
@d-ashfield rather than changing all of the existing tests that rely on a number you should add a new set of tests for the string case to ensure you didn't break the previous use case.

File: dependency-snapshot.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-frost-bunsen",
-  "version": "14.16.1",
+  "version": "14.16.2",
```
Comment (TheOtherDude) [inline dependency-snapshot.json]: You don't need to update this. `pr-bumper` will take care of it.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-frost-bunsen",
-  "version": "14.16.1",
+  "version": "14.16.2",
```
Comment (TheOtherDude) [inline package.json]: You don't need to update this. `pr-bumper` will take care of it.

**Comment by sandersky** [comment]
@d-ashfield I noticed your pull request is from another branch on the upstream, `ciena-frost`, repository. Please don't do this and instead fork the repository, making any changes on your fork. Then submit pull requests from your fork against the upstream master branch. Otherwise Travis will see the commit as a PR being merged and publish a version bump commit, which you'll notice happened on this pull request.

**Pull request closed without merging**